src/components/Splash/Splash.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+const background = require('./background.png');
```

Comment:
It definitely could have been. We don't have any layout or design settled, so I'm not sure if changing this is going to be needed. It felt weird to me to put the image in a separate file that calls this Splash page when we don't have any other image options currently.